README.md

```diff
@@ -2,8 +2,12 @@
 
 Hello!
 
-To start the quest create PR to this repository which adds a new line in this `README.md` with the text `Hello world`:
+To start the quest create PR to this repository which adds a new line in this README.md with the text Hello world:
 
 Hello world
 
 Looks good! Now let's make a Go program that will print this line for us every time we run it. Please create a PR with program code, other files if necessary and update instructions on how to run the program below:
+
+To start the program run:
+```go run main.go```
```

Comment:
+1 for markdown formatting. Small comment: if you want a separate block of code then hit an enter after the ''': https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

```
go run main.go
```

You can even do syntax highlighting:

```go
func main() {
  fmt.Println("Hello world")
}
```

If you just want to format something as code within the line `like this`, then use a single backtick before and after the block